meetings/2021-01-14.md

```diff
@@ -0,0 +1,43 @@
+# 2021-01-1478 4th meeting
+
+## Attendees
+
+* [Ben Adams](https://github.com/benaadams)
+* [Claire Novotny](https://github.com/clairernovotny)
+* [Eric Erhardt](https://github.com/eerhardt)
+* [Immo Landwerth](https://github.com/terrajobst)
+* [Jeremy Sinclair](https://github.com/snickler)
+* [John Callaway](https://github.com/ovation22)
+* [Ken Cutt](https://github.com/kencutt)
+* [Sébastien Ros](https://github.com/sebastienros)
+
+## Talking points
+
+* Discussed the proposed [survey](../docs/survey.md)
+    - The phrase "level playing field" is hard to understand
+    - We should add a similar question on trust for other big names, such as
+      Amazon, Google, component vendors
+    - We should add a question on trust regarding "open source" vs "paid source"
+    - We should add a question around money/funding for maintainers
```

Comment:
❤️ 

But here it would be interesting to dive deeper - If they fund, do they expect something in return? If so, what?

Reply:
Although, that should/can be a whole dedicated survey.

Reply:
Figuring the exact model for funding, yes. But I think the goal for the survey would be to figure how many maintainers are impacted by this.